depthai_sdk/docs/source/conf.py

```diff
@@ -0,0 +1,81 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
- regarding the logo, it's only on our `Main` docs. The API, Hardware or GUI didn't have them. I think that we should remove the logo from main docs also, since it's causing troubles with rendering, and move it to e.g. navbar - will be much easier to align and will be automatically visible on all docs subprojects
- Yes, it's a TODO, I'll add this to the docs as soon as we release the package